docs/getting-started.rst

```diff
@@ -145,7 +147,7 @@ Required columns in input data
 | immobilie_baujahr   | Construction year of dwelling                          | int   |
```

Comment:
Construction year is used in levels below. So drop the categories.

Reply:
Deleted it!